index.html

```diff
@@ -1478,6 +1476,13 @@
           a total can't be a negative number.
           </li>
         </ol>
+        <div class="note" title="No alteration of values">
+          <p>
+            One feature of this algorithm is that user agents do not alter the
```

Comment:
How about: 

```HTML
The algorithm does not alter or canonicalize the <var>amount</var>.<a>value</a>.  
For example, a user agent will not change "55" into "55.00". 
Payment handlers need to be prepared to deal with such values. 
```

Reply:
Or maybe something about being able to deal with variants (i.e., "55", "55.0", and "55.0000000000" are all be equivalent, even if presented as `US$55.00`).